src/dialogs.gs

```diff
@@ -15,7 +15,7 @@ function getDialog(file, values) {
   if(debugValue.password) delete debugValue.password;
   if(debugValue.username) delete debugValue.username;
   debug.log('Processing: %s.html with %s', file, JSON.stringify(debugValue));
-
+  // I don't think this works because it still uses values in the loop below.
```

Comment:
What might not work here?! "debugValues" is cleaned out of password/username just for the debug.log() function. In the template object it is needed, ie on the settings dialog.

Reply:
Ah ok - I see - I thought it was to prevent the password being pushed to the templates 
Apologies I read this code wrong.

I wonder if it's wise to be passing the password to the template at all, but that's a different questions